### PR TITLE
Pop Sound Theme

### LINKED ISSUE #390: Loss of sound volume notification affects accessability
User of Orca screen reading software called in with question on why we removed the sound on volume change notification.

### PR DESCRIPTION
**Pull request by isantop**
This PR adds the new Pop_OS-designed sound theme. It adds a bunch of new sounds that help to enhance the user experience, increase a11y, and provide unique auditory branding of Pop_OS.

The overall profile of the notes used within the theme were developed with guidance from Kate in design, to ensure that the sounds fit with the Pop_OS branding. 

Fixes #390 by adding a pop-branded volume change notification sound.

### CODE REVIEW

**Comment by isantop** [comment]
@pop-os/quality-assurance This is now ready for review.

**Comment by madsrh** [comment]
Hi @isantop 
Great work with the sound theme 👍 I was just wondering if it's a deliberate decision to use same sound for `software-update-urgent` and `device-removed`?

This also might be a temporary solution but the files are not symlinked but duplications (also true for `message `and `device-added`)

File: sounds/src/stereo/notification/message-new-email.oga [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/ian/Documents/Pop/gtk-theme/sounds/src/stereo/notification/message.oga
```
Comment (jackpot51) [inline sounds/src/stereo/notification/message-new-email.oga]: If these are symlinks they need to be relative

Comment (isantop) [reply sounds/src/stereo/notification/message-new-email.oga]: Good catch, thanks

Comment (isantop) [reply sounds/src/stereo/notification/message-new-email.oga]: Fixed now

**Pull request closed without merging**